src/http/AuthContext.js

```diff
@@ -10,6 +10,7 @@ class AuthContext {
     constructor() {
         this._onBehalfOf = undefined;
         this._keyLevel = config.KeyLevel.LOW;
+        this._customerInitiated = undefined;
```

Comment:
why not set it to true? It's unclear what undefined here means

Reply:
should not be true by default

Reply:
what about setting it to false then? I agree undefined seems strange